apps/src/templates/codeReviewGroups/Student.jsx

```diff
@@ -48,16 +48,55 @@ Student.propTypes = {
   index: PropTypes.number.isRequired
 };
 
+function DragHandle(props) {
+  return (
+    <div {...props} style={handleStyles.container}>
+      <div style={handleStyles.dotColumn}>
+        <span style={handleStyles.dot} />
```

Comment:
No drag icon in fontawesome presumably?

Reply:
Didn't find one that looked like ours - there is a vertical ellipsis that we could use but I figured it would be easier to just style it ourselves so we can get it to look exactly like the mocks.